src/openscm_runner/adapters/magicc7/_run_magicc_parallel.py

```diff
@@ -17,6 +19,43 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class TemporaryDirectory:
+    """A temporary directory context manager which works like
```

Comment:
pydocstyle will complain here. Run `make checks` locally to help with linter checks

Reply:
pylint is now happy at least locally.